clang/test/SemaSYCL/unique_stable_name.cpp

```diff
@@ -0,0 +1,174 @@
+// RUN: %clang_cc1 %s --std=c++17 -fsyntax-only -fsycl-is-device -Wno-unused -verify
+
+template <typename KernelName, typename KernelType>
+[[clang::sycl_kernel]] void kernel_single_task(KernelType kernelFunc) { // #1 #2 #3 #4 #5 #6 #7 #8 #9 #10 #11
```

Comment:
Labeling the kernel with the same thing on every single one defeats the purpose of the bookmarks!

Just do: #kernel_single_task, and use the same bookmark name for every one!

Reply:
I see. Then I don't have to mention the number of times such an error occured. Just using the bookmark at the call site would suffice. Ok. Thank you.